gms/feature_service.hh

```diff
@@ -82,9 +82,6 @@ public:
 
 public:
     gms::feature user_defined_functions { *this, "UDF"sv };
-    gms::feature hinted_handoff_separate_connection { *this, "HINTED_HANDOFF_SEPARATE_CONNECTION"sv };
-    gms::feature lwt { *this, "LWT"sv };
-    gms::feature per_table_partitioners { *this, "PER_TABLE_PARTITIONERS"sv };
```

Comment:
This commit seems to grandfather 3 features in one go.

Reply:
Wasn't intended.

Reply:
But how? It only grandfathers NONFROZEN_UDTS.

Reply:
Ah, the next commit does. I'll break it up.